### ISSUE TITLE
Conformers not copied correctly in renumberAtoms

### ISSUE THREAD

**Issue by JLVarjo** [rdkit/rdkit#441]
Atomic coordinates are incorrectly copied to the renumbered molecule.
Seems there's a bug in line 80:
nConf->setAtomPos(i,(*oConf)->getAtomPos(revOrder[i]));

should be:
nConf->setAtomPos(i,(*oConf)->getAtomPos(newOrder[i]));

right?

Cheers, 
Juuso

**Comment by greglandrum** [comment]
hmmm, I think the current behavior is ok:
```python
In [2]: m = Chem.MolFromSmiles('CO')
In [3]: from rdkit.Chem import AllChem
In [4]: AllChem.Compute2DCoords(m)
Out[4]: 0
In [6]: nm = Chem.RenumberAtoms(m,(1,0))
In [7]: print Chem.MolToMolBlock(m)

     RDKit          2D

  2  1  0  0  0  0  0  0  0  0999 V2000
   -0.7500    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    0.7500   -0.0000    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0
M  END

In [8]: print Chem.MolToMolBlock(nm)

     RDKit          3D

  2  1  0  0  0  0  0  0  0  0999 V2000
    0.7500   -0.0000    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
   -0.7500    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
  2  1  1  0
M  END
```
Have you seen something that makes you think it's doing something wrong?

**Comment by JLVarjo** [comment]
I think your two-atom example is not sufficient to show to bug. 

For example, my input molfile is:

   MOLECULAR MODELLING SYSTEM R1.1
    RDKit          3D

 11 10  0  0  0  0  0  0  0  0999 V2000
   1.2727   -0.2079   -0.0064 C   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0348    0.5326    0.2496 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2428   -0.3317   -0.0925 C   0  0  0  0  0  0  0  0  0  0  0  0
    2.1326    0.4172    0.2539 H   0  0  0  0  0  0  0  0  0  0  0  0
    1.3245   -1.1179    0.5988 H   0  0  0  0  0  0  0  0  0  0  0  0
    1.3671   -0.4958   -1.0587 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0854    0.8302    1.3015 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0757    2.0363   -0.7156 Cl  0  0  0  0  0  0  0  0  0  0  0  0
   -1.2460   -1.2445    0.5105 H   0  0  0  0  0  0  0  0  0  0  0  0
   -2.1755    0.2050    0.1068 H   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2366   -0.6236   -1.1479 H   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0
  2  3  1  0
  1  4  1  0
  1  5  1  0
  1  6  1  0
  2  7  1  0
  2  8  1  0
  3  9  1  0
  3 10  1  0
  3 11  1  0
M  END

.....
Then I would like all heavy atoms to come before the hydrogens:
neworder = [0, 1, 2, 7, 3, 4, 5, 6, 8, 9, 10]
Chem.rdmolops.RenumberAtoms(molecule, neworder)

and I got:


    RDKit          3D

 11 10  0  0  0  0  0  0  0  0999 V2000
    1.2727   -0.2079   -0.0064 C   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0348    0.5326    0.2496 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2428   -0.3317   -0.0925 C   0  0  0  0  0  0  0  0  0  0  0  0
    1.3245   -1.1179    0.5988 Cl  0  0  0  0  0  0  0  0  0  0  0  0
    1.3671   -0.4958   -1.0587 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0854    0.8302    1.3015 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.0757    2.0363   -0.7156 H   0  0  0  0  0  0  0  0  0  0  0  0
    2.1326    0.4172    0.2539 H   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2460   -1.2445    0.5105 H   0  0  0  0  0  0  0  0  0  0  0  0
   -2.1755    0.2050    0.1068 H   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2366   -0.6236   -1.1479 H   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0
  2  3  1  0
  1  5  1  0
  1  6  1  0
  1  7  1  0
  2  8  1  0
  2  4  1  0
  3  9  1  0
  3 10  1  0
  3 11  1  0
M  END

See e.g. the coordinates of the chlorine which are now wrong.

I also built rdkit with the above change and it started to work correctly.
